lib/texutil.js

```diff
@@ -59,8 +70,12 @@ try{ //try catch to make sure program still runs without csv file
         c++;
     }
     module.exports.paren = arr2set(paren);
-} catch (e) {}
-
+}
+catch (e) {
```

Comment:
@physikerwelt @notjagan  Yes, we are not going to use csv files anymore. We should just remove all references to that, right?